src/Spago/Config.purs

```diff
@@ -188,7 +193,7 @@ readWorkspace { maybeSelectedPackage, pureBuild } = do
       -- We try to figure out if this package has tests - look for test sources
       hasTests <- FS.exists (Path.concat [ Path.dirname path, "test" ])
       pure $ case maybeConfig of
-        Left e -> Left $ "Could not read config at path " <> path <> "\nError was: " <> e
+        Left eLines -> Left $ "Could not read config at path " <> path <> "\nError was: " <> Array.intercalate "\n" eLines
```

Comment:
The `intercalate` here is a bit of a smell because it unnecessarily cares about formatting the error message returned from above - we should return a `Docc` instead of a `Array String` from the `readConfig` function